docs/api/README.md

```diff
@@ -157,6 +157,12 @@ Allows you to replace the `matchMedia` function that Radium uses. The default is
 
 **Server**
 
+You can require `Radium` on the server / using CommonJS with:
+
+```jsx
+var Radium = require('radium');
```

Comment:
Do we not need `.default` on this import?

Reply:
Correct. That is the change with the new root `index.js` file.

Reply:
Got it. 👍